eng/SourceBuild.props

```diff
@@ -1,7 +1,6 @@
 <Project>
 
   <PropertyGroup>
-    <GitHubRepositoryName>diagnostics</GitHubRepositoryName>
```

Comment:
I think the source build leg is failing because this file doesn't include ../Directory.Build.props?